### ISSUE TITLE
Fatal error occured

### ISSUE THREAD

**Issue by TinyTiger2013** [backdrop-contrib/devel_subthemer#29]
Hello. I installed Devel Subthemer and made a subtheme today. But in "ENABLED THEMES" page, a fatal error occured when I clicked the "setting" button on the newly added subtheme. 

The error was
```Fatal error: require_once(): Failed opening required 'theme-settings.inc' (include_path='.:/usr/local/php/7.4/lib/php') in /home/example/www/examplesite/themes/axioma_subtheme/theme-settings.php on line 7```

What should I do for this?
Thanks,

**Comment by docwilmot** [comment]
Hello, sorry about that. Subthemer  doesnt seem to expect theme settings to be in a separate `.inc` file. Maybe if you copy the `theme-settings.inc` file to your new module that should fix it.

**Comment by TinyTiger2013** [comment]
Thank you for your reply. It was a shame I couldn't solve this despite your advice (I have no knowledge about php, sorry). I wait for next update! 
Thanks

**Comment by olafgrabienski** [comment]
> Subthemer doesnt seem to expect theme settings to be in a separate `.inc` file. Maybe if you copy the `theme-settings.inc` file to your new module that should fix it.

Shouldn't this be fixed by Subthemer? The parent theme in this case, Axioma, [requires](https://github.com/backdrop-contrib/axioma/blob/1.x-1.x/theme-settings.php#L7) `theme-settings.inc` in `template.php`, and according to [docs.backdropcms.org/documentation/creating-sub-themes](https://docs.backdropcms.org/documentation/creating-sub-themes) a sub-theme has to "copy the parent theme's settings". Based on that advice, I'd expect Subthemer to work with the `.inc` file.

**Comment by docwilmot** [comment]
> Shouldn't this be fixed by Subthemer? 

Not really. Keep in mind this is a tool to *help* create a subtheme. But the base theme can have any number of unusual configurations, subfolders, split things into multiple `inc` files, libraries, whatever. Subthemer can never guarantee to account for that complexity. In this particular case code that is normally in the theme settings file was split out to another file and included (could have been several includes, in an `includes` folder no less).

I could try to help with this particular case but just by advising, not changing Subthemer to account for this.

**Comment by olafgrabienski** [comment]
> Not really. Keep in mind this is a tool to *help* create a subtheme. But the base theme can have any number of unusual configurations, subfolders, split things into multiple inc files, libraries, whatever. Subthemer can never guarantee to account for that complexity.

Thanks for your reply, I see your point!

- idea: mention limitations in the README
- question: when is configuration and code organization "unusual"?

**Comment by yorkshire-pudding** [comment]
> * question: when is configuration and code organization "unusual"?

I guess if it differs from the suggested structure at https://docs.backdropcms.org/creating-themes then it could be considered unusual:

```
BACKDROP_ROOT/themes
    my_bartik
        my_bartik.info
        template.php
        theme-settings.php
        screenshot.png
        templates (directory)
            node.tpl.php
            comment.tpl.php
            maintenance-page.tpl.php    
        css (directory)
            colors.css
            (other css files)
        images (directory)
            add.png
            (other image files)
        color (directory)
            base.png
            (other color module files)
```

There should be scope to put all necessary files in that structure.
